core/arch/arm/kernel/user_ta.c

```diff
@@ -437,6 +438,68 @@ static void user_ta_dump_state(struct tee_ta_ctx *ctx)
 }
 KEEP_PAGER(user_ta_dump_state);
 
+#ifdef CFG_TA_FTRACE_SUPPORT
+static void user_ta_dump_state_buffer(struct tee_ta_ctx *ctx, char *buf,
+				      size_t *sz)
+{
+	struct user_ta_ctx *utc = to_user_ta_ctx(ctx);
+	struct user_ta_elf *elf;
+	struct vm_region *r;
+	char flags[7] = { '\0', };
```

Comment:
prefer `= { };` and add same init at line below.

Reply:
ok